### PR TITLE
feature(REPORT-333381): Bold-Reports 5.3 Release Changes

### PR DESCRIPTION
**Pull request by SundaresanGunasekaran**
<table>
<tr>
<td>
Description:
</td>
<td>
Bold-Reports 5.3 Release Changes
</td>
</tr>
<tr>
<td>
Analysis:
</td>
<td>
Analyzed about release changes
</td>
</tr>
<tr>
<td>
Reason for not identifying earlier:
</td>
<td>
It's a new feature
</td>
</tr>
<tr>
<td>
Is it a breaking issue?
</td>
<td>
No, It's a new feature
</td>
</tr>
<tr>
<td>
Solution description:
</td>
<td>
Bold-Reports 5.3 Release Changes
</td>
</tr>
<tr>
<td>
Areas affected and ensured:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Test cases:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Test bed sample location:
</td>
<td>
VS & VS Code
</td>
</tr>
</table>

### CODE REVIEW

File: NuGet/Bold.Licensing.5.3.8.nupkg
Comment (karthickthangasamy) [inline NuGet/Bold.Licensing.5.3.8.nupkg:1]: the release version files will be restored from the nuget.org, make sure the packages which are not published into public nuget.org should be committed as package.

**Review by karthickthangasamy — CHANGES_REQUESTED** [review]

File: NuGet/Bold.Licensing.5.3.8.nupkg
Comment (SundaresanGunasekaran) [reply NuGet/Bold.Licensing.5.3.8.nupkg:1]: @karthickthangasamy  - Kindly find the details
- Packages are available in nuget.org
- example - https://www.nuget.org/packages/Bold.Licensing
- I will removed the nuget packages except base logger nuget file

File: Scripts/bold-reports/v2.0/localization/l10n/ej.localetexts.ar-AE.min.js
Comment (karthickthangasamy) [inline Scripts/bold-reports/v2.0/localization/l10n/ej.localetexts.ar-AE.min.js:1]: Ensure the localization scripts required for the sample browser

File: NuGet/BoldReports.CRI.Barcode.5.2.26.nupkg
Comment (karthickthangasamy) [inline NuGet/BoldReports.CRI.Barcode.5.2.26.nupkg:1]: The new version file not committed, whether this package downloaded from nuget.org?

**Review by karthickthangasamy — CHANGES_REQUESTED** [review]

File: Scripts/bold-reports/v2.0/localization/l10n/ej.localetexts.ar-AE.min.js
Comment (SundaresanGunasekaran) [reply Scripts/bold-reports/v2.0/localization/l10n/ej.localetexts.ar-AE.min.js:1]: we need default localization en-US files only, so I have removed the other locale files

File: NuGet/BoldReports.CRI.Barcode.5.2.26.nupkg
Comment (SundaresanGunasekaran) [reply NuGet/BoldReports.CRI.Barcode.5.2.26.nupkg:1]: Removed

**Review by karthickthangasamy — APPROVED** [review]

**Pull request merged**